ui/src/alerting/components/CheckEOHeader.tsx

```diff
@@ -98,6 +98,10 @@ const CheckEOHeader: FC<Props> = ({
   }
 
   const {singleField, singleAggregateFunc} = isDraftQueryAlertable(draftQueries)
+  const oneOrMoreThresholds =
+    check.type === 'threshold'
+      ? check.thresholds && !!check.thresholds.length
+      : null
```

Comment:
i think this should be `false`, not `null`. `CheckEOSaveButton` is expecting `oneOrMoreThresholds` to be a boolean and making this explicitly `false` will let us not rely on JS's sometimes confusing type casting. what do you think?